docs/OnnxReleases.md

```diff
@@ -2,25 +2,43 @@ The ONNX project, going forward, will plan to release roughly on a two month cad
 
 Next expected release: May 15th 
 
-Release Checklist: 
+## Preparation
 
-* Install Twine, a utility tool to interact with pypi. Do  - ``pip install twine``
-* Get hold of the username and password for the ‘onnx’ pypi account.
-* Pick a release tag for the new release through mutual consent – Slack channel for Releases (https://lfaifoundation.slack.com/archives/C018VGGJUGK)
+* Install Twine, a utility tool to interact with PyPI. Do  - ``pip install twine``
+* Get hold of the username and password for the ‘onnx’ PyPI account. Release Manager should obtain this information from last Release Manager or certain private repo under onnx.
+* Pick a release tag (v.1.X.X) for the new release through mutual consent – Slack channel for Releases (https://lfaifoundation.slack.com/archives/C018VGGJUGK)
 * Prepare a change log for the release – 
     * ``git log --pretty=format:"%h - %s" <tag of the previous release>...<new tag>``
     * And draft a new release statement - https://github.com/onnx/onnx/releases listing out the new features and bug fixes, and potential changes being introduced in the release.
-* Checkout the release tag in a clean branch on your local repo. Make sure all tests pass on that branch.
-* Make sure that the Release version number information is up-to-date in the following places:
+* Create a release branch (rel-1.X.X) from master. Checkout the release tag in a clean branch on your local repo. Make sure all tests pass on that branch.
+* In the release branch, make sure that the Release version number information is up-to-date in the following places:
```

Comment:
updating release version happens on master before release branch is created right? Can you move this step above Create release branch and mention this needs to happen in master

Reply:
I meant can you move updating release number in all files listed below newar line 11 (before creating release branch)